data/supertypes.yml

```diff
@@ -381,3 +380,192 @@ government_document_supertype:
     - id: government-responses
       document_types:
         - government_response
+
+content_purpose_subgroup:
```

Comment:
It might be worth marking `content_purpose_document_supertype` as deprecated so that there's less confusion while it's being removed.

Reply:
Cool, I have marked this is deprecated.